src/test/java/it/gov/pagopa/reward/notification/controller/NotificationControllerImplTest.java

```diff
@@ -0,0 +1,99 @@
+import org.springframework.http.HttpStatus;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@WebFluxTest(controllers = {RewardExportsDTO.class})
+@Import(NotificationControllerImpl.class)
+class NotificationControllerImplTest {
+
+    @MockBean
+    OrganizationExportsServiceImpl organizationExportsService;
+
+    @MockBean
+    ErrorNotifierService errorNotifierService;
+
+    @Autowired
+    protected WebTestClient webClient;
+
+    @Test
+    void testGetExportsOk() {
+        RewardExportsDTO rewardExportsDTOMock = RewardExportsDTOFaker.mockInstance(1);
+        Mockito.when(organizationExportsService.findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any()))
+                .thenReturn(Flux.just(rewardExportsDTOMock));
+
+        webClient.get()
+                .uri(uriBuilder -> uriBuilder.path("/idpay/organization/{organizationId}/initiative/{initiativeId}/exports")
+                        .build(rewardExportsDTOMock.getOrganizationId(), rewardExportsDTOMock.getInitiativeId()))
+                .exchange()
+                .expectStatus().isOk()
+                .expectBody(RewardExportsDTO.class).isEqualTo(rewardExportsDTOMock);
+
+        Mockito.verify(organizationExportsService, Mockito.times(1)).findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any());
+    }
+
+    @Test
+    void testGetExportsNotFound() {
+        RewardExportsDTO rewardExportsDTOMock = RewardExportsDTOFaker.mockInstance(1);
+        Mockito.when(organizationExportsService.findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any()))
+                .thenReturn(Flux.error(new ClientExceptionNoBody(HttpStatus.NOT_FOUND)));
+
+        webClient.get()
+                .uri(uriBuilder -> uriBuilder.path("/idpay/organization/{organizationId}/initiative/{initiativeId}/exports")
+                        .build(rewardExportsDTOMock.getOrganizationId(), rewardExportsDTOMock.getInitiativeId()))
+                .exchange()
+                .expectStatus().isNotFound();
+
+        Mockito.verify(organizationExportsService, Mockito.times(1)).findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any());
+    }
+
+    @Test
+    void testGetExportsCountOk() {
+        Long count = 1L;
+        Mockito.when(organizationExportsService.countAll(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any()))
+                .thenReturn(Mono.just(count));
+
+        webClient.get()
+                .uri(uriBuilder -> uriBuilder.path("/idpay/organization/{organizationId}/initiative/{initiativeId}/exports/count")
+                        .build("ORG", "INTV"))
+                .exchange()
+                .expectStatus().isOk()
+                .expectBody(Long.class).isEqualTo(count);
+
+        Mockito.verify(organizationExportsService, Mockito.times(1)).findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any());
```

Comment:
don't use Mockito.any to verify, but the expected values, in order to verify that the controller has successfully passed them